### PR TITLE
feat/balance-sheet-page

### PR DESCRIPTION
**Pull request by ivangarcilazo**

### CODE REVIEW

**Comment by mohitkhatri06** [comment]
Hi Ivan for the following table lets make it with the same color as in the figma because different colors can reduce the readability.
image

File: src/components/Charts/AssetsLiabilitiesChart.jsx [hunk 1]
```diff
@@ -177,9 +135,10 @@ export const AssetsLiabilitiesChart = (props) => {
...
-                        layout='vertical'
-                        verticalAlign='top'
-                        align='right'
+                        width={'100%'}
+                        layout='horizontal'
+                        verticalAlign='bottom'
+                        align='center'
                         payload={pieData.map((item, index) => ({
                            id: item.name,
                            type: 'square',
```
Comment (mohitkhatri06) [inline src/components/Charts/AssetsLiabilitiesChart.jsx:144]: change this to circle as in figma  legend are in the circle shape.

**Comment by ivangarcilazo** [comment]
Sure! Thanks for the comments.

**Comment by mohitkhatri06** [comment]
after page refresh of the balance page it is not making any GET call for "loan" and "physical _assets"

**Comment by ivangarcilazo** [comment]
Sorry Mohit! Fixed it!

**Review by mohitkhatri06 — APPROVED** [review]

**Pull request merged**